Conversion-Class Temperatura

```diff
@@ -0,0 +1,18 @@
+package com.company;
+
+public class Temperatura {
+
+    public double temperatura;
+    public void setTemperatura(double temperaturaZilei) {
+        this.temperatura = temperaturaZilei;
+    }
+    public double getTemperatura(){
+        return temperatura;
+    }
+
+    public void temperatura (double c, double f) {
+        double celsius = 0;
+        double fahrenheit = 70;
+        double temperatura = 5/9*(f-32);
```

Comment:
If you want to convert the `temperatura`, you can get the full use of your getter and setter defined above. No need of any arguments to be passed. If can be something like this: 
`this.setTemperatura(5/9*(this.temperatura - 32))`